convert2rhel/main.py

```diff
@@ -151,6 +151,10 @@ def pre_ponr_conversion():
     # check if user pass some repo to both disablerepo and enablerepo options
     pkghandler.has_duplicate_repos_across_disablerepo_enablerepo_options()
 
+    # checking if /mnt and /sys are read-write
+    loggerinst.task("Convert: Checking /mnt and /sys are read-write")
+    utils.check_if_mount_is_readonly()
```

Comment:
I created a special python module checks.py (see #193 ) for doing this kind of checks. Could you please grab it into your MR, so we'll have your check also in checks.py? WDYT?

Reply:
They're not related, so whichever get merged first should adhere to the other - not create dependencies. Either PR can be pushed back due to reviews, so keep stacked PRs to a minimum.

Reply:
@SpyTec It is not stacking MRs, but synchronizing on the solution to implement different kinds of checks. I am reviewing this MR and I see that Fellipe implement check. I know that I also do this. My intention is to synchronize with the team on doing checks.

I would call this as weak dependency.

Reply:
No need to merge #193. We can merge this one and then, upon rebasing #193, @ZhukovGreen you can just move this check to the new checks.py.

Reply:
@bocekm Definitely no need to wait for #193